webapp/src/modules/rental/selectors.ts

```diff
@@ -23,3 +24,19 @@ export const isAcceptingRental = (state: RootState) =>
   isLoadingType(getLoading(state), ACCEPT_RENTAL_LISTING_REQUEST)
 export const isSubmittingTransaction = (state: RootState) =>
   getState(state).isSubmittingTransaction
+
+export const getRentStatus = (state: RootState) => {
```

Comment:
What do you think about renaming the function to be in accordance to the name of the operation? The idea is to avoid confusion with the rental status per se (the one that's on the server).
```suggestion
export const getRentConfirmationStatus = (state: RootState) => {
```